### PR TITLE
Add Row layout

### PR DESCRIPTION
**Pull request by mfrachet**
Add a row layout component.

I'm not a huge fan of this one since it may break responsiveness in the future. But since I don't find any good solutions to handle row base layout, I'm opening this one.

It only contains a minimalist set of CSS rules. If we feel the need to add more, let's add them in another pr 😊

### CODE REVIEW

File: packages/strapi-design-system/src/Row/Row.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Row.defaultProps = {
+  reverse: false,
+  justifyContent: undefined,
+  alignItems: 'center',
+  wrap: undefined,
+};
+
+Row.propTypes = {
+  reverse: PropTypes.bool,
```
Comment (soupette) [inline packages/strapi-design-system/src/Row/Row.js:20]: Why this is a boolean?

File: packages/strapi-design-system/src/Row/Row.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  justifyContent: PropTypes.string,
```
Comment (soupette) [inline packages/strapi-design-system/src/Row/Row.js:21]: Can you order the propTypes alphabetically ?

File: packages/strapi-design-system/src/Row/Row.js (hunk 1 above)
Comment (mfrachet) [reply packages/strapi-design-system/src/Row/Row.js:20]: Cause I failed 😆

File: packages/strapi-design-system/src/Row/Row.js (hunk 2 above)
Comment (mfrachet) [reply packages/strapi-design-system/src/Row/Row.js:21]: come oooon, right, I can do that 😆

Comment (soupette) [reply packages/strapi-design-system/src/Row/Row.js]: Maybe we could add this https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-prop-types.md ?

**Review by HichamELBSI — APPROVED** [review]
LGTM

File: packages/strapi-design-system/src/Row/Row.js (hunk 2 above)
Comment (mfrachet) [reply packages/strapi-design-system/src/Row/Row.js]: Yes, why not 😊 . Can you do it?

**Pull request merged**